### PR TITLE
Fix Wagtail preview bug and import script

### LINKED ISSUE #93: repair preview functionality in wagtail admin
previewing pages via the wagtail admin currently leads to a couple of different exceptions.

to reproduce, create a page in the wagtail cms, click "save draft", then try to preview the page. you should see a 500 error.

sentry exception #1: https://sentry.io/organizations/datamade/issues/1249526277/?referrer=slack

```
AttributeError: 'WSGIRequest' object has no attribute 'site'
  File "django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 156, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "django/core/handlers/base.py", line 154, in _get_response
    response = response.render()
  File "django/template/response.py", line 106, in render
    self.content = self.rendered_content
  File "django/template/response.py", line 83, in rendered_content
    content = template.render(context, self._request)
  File "django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "django/template/base.py", line 171, in render
    return self._render(context)
  File "django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "django/template/base.py", line 937, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 904, in render_annotated
    return self.render(context)
  File "django/template/library.py", line 192, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "committeeoversightapp/templatetags/cms_tags.py", line 9, in get_site_root
    return context['request'].site.root_page
```

sentry exception #2: https://sentry.io/organizations/datamade/issues/1249526276/?project=1758781&referrer=slack

```
AttributeError: 'NoneType' object has no attribute 'lower'
  File "django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "django/utils/deprecation.py", line 90, in __call__
    response = self.process_request(request)
  File "django/middleware/common.py", line 47, in process_request
    host = request.get_host()
  File "django/http/request.py", line 97, in get_host
    domain, port = split_domain_port(host)
  File "django/http/request.py", line 554, in split_domain_port
    host = host.lower()
```

### PR DESCRIPTION
**Pull request by beamalsky**
## Overview

Closes #93. This PR fixes a bug with draft previewing for local development. After some investigation, I discovered that the `Site` model should have a `pk` of 1 but had been mistakenly exported with a `pk` of 2. This came from a deeper problem, where the `load_cms_content` script was not cleaning up the Wagtail database before loading in initial data. This has led to unwanted primary key collisions and general messiness, and broken image links. Since we'll only use the script to move/recreate site data entirely, it's enough for this site to delete Wagtail models before importing `initial_cms_content.json`. This PR also fixes those broken images!

In the process of troubleshooting this, I updated Wagtail to the newest version (2.6.2).

## Testing Instructions

* Delete your volume: `docker-compose down --volumes`
* Start the site back up: `docker-compose up`
* Open a new window and load `hearings.dump`:  
`docker-compose exec postgres pg_restore -C -j4 --no-owner -U postgres -d hearings /app/hearings.dump`
* Run migrations: `docker-compose exec app python manage.py migrate`
* Load CMS content: `docker-compose run --rm app python manage.py load_cms_content`
* Load sample committee ratings: `docker-compose run --rm app python manage.py load_committeeratings`
* Log into the site at `http://localhost:8000/cms`
* Create a new page and make sure draft preview functionality works
* Check on the images and make sure they're displaying properly. You should see 3 images from Lugar's website: 
Screen Shot 2019-10-08 at 4 26 37 PM

### CODE REVIEW

**Comment by hancush** [comment]
@beamalsky Static page previews work, but I'm still getting this exception when trying to preview a draft of a committee detail page:

```
Template error:
In template /app/committeeoversightapp/templates/base.html, error at line 38
   'WSGIRequest' object has no attribute 'site'
   28 :     <link rel="stylesheet" href="{% static 'css/select2.min.css' %}">
   29 :     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
   30 :     <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:300,400|Roboto:300,400,400i,500,700,700i&display=swap" rel="stylesheet">
   31 : 
   32 :     <title>Committee Oversight Data Tool</title>
   33 :   </head>
   34 : 
   35 :   <body>
   36 : 
   37 :     {% block menu %}
   38 :        {% get_site_root as site_root %} 
   39 :       {% top_menu parent=site_root calling_page=self %}
   40 :     {% endblock %}
   41 : 
   42 :     <div class="container">
   43 :       <div class="row justify-content-center">
   44 :           {% wagtailuserbar %}
   45 :           {% block content %}
   46 :           {% endblock %}
   47 :       </div>
   48 :     </div>


Traceback:

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  156.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py" in _get_response
  154.                 response = response.render()

File "/usr/local/lib/python3.7/site-packages/django/template/response.py" in render
  106.             self.content = self.rendered_content

File "/usr/local/lib/python3.7/site-packages/django/template/response.py" in rendered_content
  83.         content = template.render(context, self._request)

File "/usr/local/lib/python3.7/site-packages/django/template/backends/django.py" in render
  61.             return self.template.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render
  171.                     return self._render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in _render
  163.         return self.nodelist.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py" in render
  150.             return compiled_parent._render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in _render
  163.         return self.nodelist.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/loader_tags.py" in render
  62.                 result = block.nodelist.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render
  937.                 bit = node.render_annotated(context)

File "/usr/local/lib/python3.7/site-packages/django/template/base.py" in render_annotated
  904.             return self.render(context)

File "/usr/local/lib/python3.7/site-packages/django/template/library.py" in render
  192.         output = self.func(*resolved_args, **resolved_kwargs)

File "/app/committeeoversightapp/templatetags/cms_tags.py" in get_site_root
  9.     return context['request'].site.root_page

Exception Type: AttributeError at /cms/pages/5/edit/preview/
Exception Value: 'WSGIRequest' object has no attribute 'site'
```

I made sure I had the right branch checked out and pulled, and followed the testing instructions. (The only change I made was that I ran `docker-compose up --build -d` in order to capture the Wagtail upgrade.)

**Comment by beamalsky** [comment]
@hancush I'm kind of going in circles here—I'm not sure why Wagtail isn't including a `site` attribute in preview requests for custom models, but it's not. This causes errors in the top menu cms tags introduced in #84. 

I pushed a bandaid solution in [the last commit](https://github.com/datamade/committee-oversight/pull/94/commits/c84fac24f2eab6a7fa455f2af87108a08138c9f5), which fixes the error: if the `site` isn't present, it grabs the first `site` object available. Since this app should only ever have one site, and previews are low stakes, this is a workable but unsatisfying solution. If you have time to take a look at this more deeply, I'd appreciate your insight!

**Comment by beamalsky** [comment]
Update! @hancush figured out that `get_url_parts` was the culprit.

File: committeeoversightapp/models.py [hunk 1]
```diff
@@ -202,12 +202,13 @@ def title_field(self):
 
     body = RichTextField()
 
-    def get_url_parts(self, *args, **kwargs):
-        '''
-        Route detail pages by their slug.
-        https://docs.wagtail.io/en/v2.6.1/topics/pages.html#obtaining-urls-for-page-instances
-        '''
-        return ('', '', '/' + self.slug)
+    # def get_url_parts(self, *args, **kwargs):
```
Comment (hancush) [inline committeeoversightapp/models.py]: want to remove this commented code?

File: committeeoversightapp/templatetags/cms_tags.py [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 register = template.Library()
 
+from wagtail.core.models import Site
```
Comment (hancush) [inline committeeoversightapp/templatetags/cms_tags.py]: do you need this import?

Comment (beamalsky) [reply committeeoversightapp/templatetags/cms_tags.py]: nope!

File: committeeoversightapp/models.py (hunk 1 above)
Comment (beamalsky) [reply committeeoversightapp/models.py]: yep! hm idk why I did this!

**Pull request merged**